tests/features/with_test.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import annotations
+
+import sys
+
+import pytest
+
+from add_trailing_comma._main import _fix_src
+from add_trailing_comma._main import main
+
+
+@pytest.mark.parametrize(
+    ('src', 'plus39'),
+    (
+        pytest.param(
+            'from threading import Lock\n'
+            'with(Lock() as l):\n'
```

Comment:
none of these tests look like how I would expect someone to write code -- it's useful to have one test without whitespace after the `with` token but unless there's a good reason most of the tests should have a space after `with` (it's not a function after all)